### ISSUE TITLE
Screen reader is not reading the action performed when we check/uncheck the select all checkbox

### ISSUE THREAD

**Issue by sandeepnmenon** [OfficeDev/office-ui-fabric-react#10519]
### Environment Information

- **Package version(s)**: (6.182.1)
- **Browser and OS versions**: (Edge/Chrome and Windows 10)

> Note that our screen reader support is limited to Edge + Narrator. Issues that do not reproduce in Edge + Narrator, and aren't caused by obvious invalid aria values, should be filed with the respective screen reading software, not the Fabric repo.

### Describe the issue:

Screen reader(Narrator/NVDA) is not reading the action performed when we check/uncheck the select all checkbox

### Please provide a reproduction of the issue in a codepen:
https://codepen.io/sandeepnmenon/pen/NWKOgYo?&editable=true

#### Actual behavior:

Narrator/NVDA is not reading the action performed when we check/uncheck the "select all" checkbox

#### Expected behavior:

Narrator/NVDA should read the action performed when we check/uncheck the "select all" checkbox

**Comment by aneeshack4** [comment]
This does not repro on Edge + Narrator.
![image](https://user-images.githubusercontent.com/4161791/65555378-5feebc80-dee1-11e9-9e0e-05a589c64bfc.png)
@sandeepnmenon are you sure you're using the latest version of Fabric (v7)? Also if this is a NVDA specific issue, please file an issue with them as they have a [public repo]( https://github.com/nvaccess/nvda).

**Comment by sandeepnmenon** [comment]
@aneeshack4 Yes it reads checked/unchecked on the select all button and
select/unselected for the items.
The message check/unchecked is not clear if we are selecting all items or not. There is no indication that we are selecting all items in the list.
That is the bug
